src/statemachine/statemachine_coroutine.js

```diff
@@ -0,0 +1,225 @@
+pc.extend(pc, (function () {
```

Comment:
It's up to you. You should be able to update the files and just update the PR with no problems though.